src/vue/modules/verification/general-form/store/index.js

```diff
@@ -229,36 +227,36 @@ const getters = {
     if (state.isAccredited) {
       if (state.kycProofOfInvestor) {
         blobData.documents[DOCUMENT_TYPES.kycProofOfInvestor] =
-          state.documents.proofOfInvestor.getDetailsForSave()
+          state.documents.proofOfInvestor.toJSON()
```

Comment:
no need to call `.toJSON` explicitly. It is automatically called by `JSON.stringify`

Reply:
`verification/general-form/*` is to be removed by me  
The application needs some preparations to omit explicit calls of `toJSON()`. Let it be for now